src/app/view/components/toolbar-user/toolbar-user.html

```diff
@@ -7,6 +7,7 @@
     (open)="iframeOverlayService.show()"
     (close)="iframeOverlayService.hide()">
     <div class="toolbar-user__content">
+        <dot-gravatar [size]="70" [email]="auth.user.emailAddress || auth.loginAsUser.emailAddress"></dot-gravatar>
```

Comment:
I think can have the param ```size``` without the ```[]``` because it's not a variable what you are passing.